src/dt/perspectives/machine_ethics/calculate_score.py

```diff
@@ -26,6 +24,13 @@ def main():
         print(f"Dataset: {dataset_name}")
         option = OPTION_DIC[dataset_name]
 
+        # Get all model names
+        cur_dir = os.path.join(base_dir, dataset_name)
+        while glob.glob(f'{cur_dir}/*.json') == []:
+            cur_dir = os.path.join(cur_dir, '**')
+        model_list = glob.glob(f'{cur_dir}/*.json')
```

Comment:
Can you double-check this? `glob` defaults to `recursive=False`. See: https://docs.python.org/3/library/glob.html#glob.glob

Reply:
Done, thanks!